src/entity/entities.tsx

```diff
@@ -0,0 +1,24 @@
+import { Role } from '../enum/role'
+
+export const user = (username: string,
+                    password: string,
+                    firstname: string,
+                    lastname: string,
+                    phoneNumber: string,
+                    role: Role,
+                    products: Object[],
+                    histories: Object[]) => ({
+  username,
+  password,
+  firstname,
+  lastname,
+  phoneNumber,
+  role,
+  products,
+  histories
+})
```

Comment:
Just curious cause I'm not quite familiar with this kind of format, Do you think my suggestion is easier to read or we should keep it this way? BTW, both of them are fine with me.

```suggestion
export const user = (
  username: string,
  password: string,
  firstname: string,
  lastname: string,
  phoneNumber: string,
  role: Role,
  products: Object[],
  histories: Object[]
) => ({
  username,
  password,
  firstname,
  lastname,
  phoneNumber,
  role,
  products,
  histories
})
```